resources/swiftlint.yml

```diff
@@ -38,3 +38,10 @@ custom_rules:
     regex: "@objcMembers"
     message: "Explicitly use @objc on each member you want to expose to Objective-C"
     severity: error
+  no_direct_standard_out:
+    name: "Writing directly to standard out disallowed"
+    regex: "(\\bprint|\\bdebugPrint|\\bdump|Swift\\.print|Swift\\.debugPrint|Swift\\.dump)\\s*\\("
```

Comment:
TIL a space before the `(` is valid.
<img width="947" alt="Screen Shot 2021-04-22 at 8 58 21 PM" src="https://user-images.githubusercontent.com/1791049/115816304-80c50f80-a3ad-11eb-940a-e6e1b08bdd74.png">